browser-test/src/trusted_intermediary.test.ts

```diff
@@ -305,7 +325,7 @@ test.describe('Trusted intermediaries', () => {
       lastName: 'last',
       dobDate: '2023-07-11',
     }
-    await page.getByRole('button', {name: 'Add new client'}).click()
+    await page.locator('#add-new-client').click()
```

Comment:
Nitpick: Is it worth defining a helper method for clicking the "Add new client" button since we do it in multiple tests?

Reply:
@caitlinshk summoned me on this PR so everything I say is her fault.

Don't change these locators. Playwright best practices are to use `page.getByRole` first, other `page.getByXxx` methods second, and only fallback to element id as a last resort.

The reason being that Playwright looks up elements based on w3c accessibility name which is akin to how assistive technologies see the elements and it makes for less fragile tests.

Reply:
I tried getByRole() but this is not a button but an ATag. I am not sure how to get it. I checked https://playwright.dev/docs/locators#locate-by-role  doesn't seem to have it. @gwendolyngoetz and @caitlinshk  - Do you know?

Reply:
If it's an `<a>` you can use the role 'link'

Reply:
Thanks @gwendolyngoetz